pkg/apis/hive/v1/clusterdeprovision_types.go

```diff
@@ -21,6 +21,9 @@ type ClusterDeprovisionSpec struct {
 type ClusterDeprovisionStatus struct {
 	// Completed is true when the uninstall has completed successfully
 	Completed bool `json:"completed,omitempty"`
+
+	// BlockedResources is a list of cloud resources that the deprovision has not been able to delete
+	BlockedResources []string `json:"blockedResources,omitempty"`
```

Comment:
How are things looking for the other cloud providers, will a flat list of strings be sufficient as far as we can see? Wondering if we should go with something that lets us store a little more data or not.

Reply:
That's a good idea. I can change `BlockedResources` to a struct that for now has nothing but a name field.